baton/hub.go

```diff
@@ -2,94 +2,86 @@ package baton
 
 import (
 	"encoding/json"
-	"fmt"
 	"log"
-)
 
-type command struct {
-	Module string      `json:"module"`
-	Call   string      `json:"call"`
-	ID     string      `json:"id"`
-	Body   interface{} `json:"body"`
-}
+	"github.com/hackedu/maestro/baton/commands"
+)
 
 type rawMsg struct {
 	conn conn
 	data []byte
 }
 
-type msg struct {
-	conn conn
-	cmd  command
-}
-
 type hub struct {
-	conns      map[conn]struct{}
+	conns      map[conn][]string
+	ids        map[string]conn
```

Comment:
A potentially clearer way to do this would be to create a new type along the lines of:

    type CommandID string

And then make this type `map[CommandID]conn`. Thoughts?